sdk/cosmosdb/cosmos/package.json

```diff
@@ -94,6 +94,7 @@
     "@azure/core-auth": "^1.3.0",
     "@azure/core-rest-pipeline": "^1.2.0",
     "@azure/core-tracing": "^1.0.0",
+    "@azure/identity": "^2.0.1",
```

Comment:
Why do you need to depend on `@azure/identity` directly? The TokenCredential interface from core-auth should be enough unless you need to use concrete credential implementations.  Customers who don't use AAD auth shouldn't need this dependency.